css/main.css

```diff
@@ -559,4 +553,120 @@ textarea {
     width: 3vw;
     height: 32px;
   }
-}
\ No newline at end of file
+}
+
+/* Mobile Menu styling */
+
+.header {
+  background-color: #3c3a39;
+  color: #fff5e1;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem 1.5rem;
+}
+
+.hamburger {
+  display: none;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #67798e;
+}
+
+.nav-menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav-item {
+  margin-left: 5rem;
+  border-bottom: 1px solid #fff5e1;
+}
+
+.nav-link {
+  font-size: 1.6rem;
+  font-weight: 400;
+  color: #fff5e1;
+}
+
+.nav-link:hover {
+  color: #f55800;
+}
+
+.nav-logo {
+  font-size: 2.1rem;
+  font-weight: 500;
+  color: #fff5e1;
+}
+
+@media only screen and (max-width: 768px) {
+  * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  html {
+    font-size: 62.5%;
+    font-family: 'Roboto', sans-serif;
+  }
+
+  li {
+    list-style: none;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  .nav-menu {
+    position: fixed;
+    left: -100%;
+    top: 5rem;
+    flex-direction: column;
+    background-color: #3c3a39;
+    opacity: 0.9;
```

Comment:
- [ ] While working on the previous review, opacity was set to 0.9, making the background of the mobile menu transparent, kindly fix it to make it match with the Figma design.